guides/source/command_line.md

```diff
@@ -376,37 +376,37 @@ $ bin/rails destroy model Oops
       remove      test/fixtures/oops.yml
 ```
 
-Rake
-----
+bin/rails
+---------
 
-Rake is Ruby Make, a standalone Ruby utility that replaces the Unix utility 'make', and uses a 'Rakefile' and `.rake` files to build up a list of tasks. In Rails, Rake is used for common administration tasks, especially sophisticated ones that build off of each other.
+Rake is Ruby Make, a standalone Ruby utility that replaces the Unix utility 'make', and uses a 'Rakefile' and `.rake` files to build up a list of tasks. In Rails, Rake is used for common administration tasks, especially sophisticated ones that build off of each other. Since Rails 5.0+ rake commands implemented in Rails Framework itself. We prefer to use bin/rails instead of rake.
 
-You can get a list of Rake tasks available to you, which will often depend on your current directory, by typing `rake --tasks`. Each task has a description, and should help you find the thing you need.
+You can get a list of bin/rails tasks available to you, which will often depend on your current directory, by typing `bin/rails --tasks`. Each task has a description, and should help you find the thing you need.
```

Comment:
Yes, but maybe we should just show an abbreviated list. I don't want it to grow stale whenever we add a command.

(Also don't add that `a:loc` task to our documentation 😁)